forge-gui/res/cardsfolder/upcoming/braided_net_braided_quipu.txt

```diff
@@ -0,0 +1,24 @@
+Name:Braided Net
+ManaCost:2 U
+Types:Artifact
+K:etbCounter:NET:3
+A:AB$ Tap | Cost$ T SubCounter<1/NET> | ValidTgts$ Permanent.Other+nonLand | TgtPrompt$ Select another target nonland permanent | SubAbility$ DBEffect | SpellDescription$ Tap another target nonland permanent.
+SVar:DBEffect:DB$ Effect | RememberObjects$ Targeted | StaticAbilities$ CantActivate | ForgetOnMoved$ Battlefield | Triggers$ UntapTrig | SpellDescription$ Its activated abilities can't be activated for as long as it remains tapped.
```

Comment:
`ForgetOnMoved$ Battlefield | Triggers$ UntapTrig` :arrow_right: `Duration$ UntilTargetedUntaps`

Reply:
I thought we had something better for this. Should have just asked (or dug harder to find it)

Reply:
To be fair I think I only recently added that